app/controllers/v1/users_controller.rb

```diff
@@ -34,7 +34,7 @@ def destroy
 
   def user_params
     # whitelist params
-    params.permit(:username)
+    params.permit(:username, :email, :password)
```

Comment:
Here I added email and password, as only username was allowed and new users were not created.